Sources/App/Extensions/GitHubService+IssueComment.swift

```diff
@@ -0,0 +1,116 @@
+import Vapor
+import Stevenson
+
+struct PingAction: Content {
+    let zen: String
+}
+
+struct CommentAction: Content {
+    let action: String
+    let comment: Comment
+    let issue: Issue
+    let repository: Repository
+
+    struct Comment: Content {
+        let body: String
+    }
+
+    struct Issue: Content {
+        let number: Int
+    }
+
+    struct Repository: Content {
+        let full_name: String
```

Comment:
Is it with underscore because of some convention?

Reply:
this is copy-paste, git didn't pick up renaming the file :/ I think we're avoiding adding custom decoder for the object this way

Reply:
Ok I see it was like that before.